src/SPC/builder/unix/library/libtiff.php

```diff
@@ -24,17 +29,21 @@ protected function build(): void
                 "--with-zlib-lib-dir={$this->getLibDir()}",
                 // libjpeg deps
                 '--enable-jpeg',
-                '--disable-old-jpeg',
-                '--disable-jpeg12',
                 "--with-jpeg-include-dir={$this->getIncludeDir()}",
                 "--with-jpeg-lib-dir={$this->getLibDir()}",
-                // We disabled lzma, zstd, webp, libdeflate by default to reduce the size of the binary
-                '--disable-lzma',
-                '--disable-zstd',
-                '--disable-webp',
+                '--disable-old-jpeg',
+                '--disable-jpeg12',
                 '--disable-libdeflate',
+                '--disable-tools',
+                '--disable-contrib',
                 '--disable-cxx',
+                '--without-x',
             )
+            ->optionalLib('lerc', '--enable-lerc', '--disable-lerc')
+            ->optionalLib('zstd', '--enable-zstd', '--disable-zstd')
+            ->optionalLib('webp', '--enable-webp', '--disable-webp')
+            ->optionalLib('xz', '--enable-lzma', '--disable-lzma')
+            ->optionalLib('jbig', '--enable-jbig', '--disable-jbig')
```

Comment:
If libtiff contain optional libs, we cannot provide pre-built libs for libtiff anymore. That is, we need to remove `libtiff.provide-pre-built` line in `source.json`.